### PR TITLE
Edits for the first revision

### PR DESCRIPTION
**Pull request by drvinceknight**
This is a PR in to `plos-first-revision-master` which is a clean fork of `plos` (the one we submitted) to allow us to easily modify and discuss the modification.

Each commit should be relatively self explanatory and aims to address a specific comment by the reviewer.

One particularity 3049b2d and b01d282 are essentially just to make the journal system happy by changing the format of all the images to eps (and naming them `Fig2.eps` ...).

### CODE REVIEW

File: response_to_reviewers.tex [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+This is a well made point by the referee: ensuring the reader can understand
+these is vital to the paper.
+We have clarified the description of the strategies:
+
+\begin{itemize}
+    \item Highlight that the appendix contains references for each strategy
+        including pointing to a recently published paper that goes in to details
+        about the structure of these strategies.
+    \item A more formal definition of a finite state machine explaining how they
```
Comment (marcharper) [inline response_to_reviewers.tex:54]: Let's make sure we have the appropriate Ashlock et al references since they go into much more detail.

**Review by marcharper — APPROVED** [review]

File: main.tex [hunk 2]
```diff
@@ -569,20 +574,28 @@ \subsection*{Strategies considered}\label{sec:strategies}
...
+One particular family of strategies that has been studied in the literature are
+called finite state machines. These mathematical models consist of states and
+and responses to actions from one state to another. In the context of the IPD, a
+finite state machine, is a mapping from states and opponent actions to states
+and an action. For further details on this type of strategy, the reader is
+referred to~\cite{Ashlock2006, Harper2017}.
+
+There are three further strategies trained specifically for this
+study; Trained FSM 1, 2, and 3 (TF1 - TF3). These are finite state
+machines of 16, 16, and 8 states respectively.  These are shown in
```
Comment (Nikoleta-v3) [inline main.tex]: delete space

File: main.tex [hunk 3]
```diff
@@ -331,13 +331,21 @@ \section*{Introduction}
...
+     mechanism.
  \item Strong invaders are generally cooperative strategies that do not defect
  first but retaliate to varying degrees of intensity against strategies that defect.
  \item Strategies evolved to maximize their total payoff can be strong invaders
  and achieve mutual cooperation with many other strategies.
 \end{enumerate}
 
+The notion of a handshake has been described previously in~\cite{Robson1990} and
+corresponds to idea that an individual exhibits behaviour that starts with a
+recognisable pattern. This in turn allows them to identify memers of their own
```
Comment (Nikoleta-v3) [inline main.tex]: members*

File: main.tex [hunk 4]
```diff
@@ -593,13 +594,14 @@ \subsection*{Strategies considered}\label{sec:strategies}
 
 One particular family of strategies that has been studied in the literature are
 called finite state machines. These mathematical models consist of states and
-and responses to actions from one state to another. In the context of the IPD, a
-finite state machine, is a mapping from states and opponent actions to states
-and an action. For further details on this type of strategy, the reader is
-referred to~\cite{Ashlock2006, Harper2017}.
+and responses to actions which indicate a next state given an action.  In the
```
Comment (Nikoleta-v3) [inline main.tex]: delete space

File: main.tex [hunk 5]
```diff
@@ -593,13 +594,14 @@ \subsection*{Strategies considered}\label{sec:strategies}
 
 One particular family of strategies that has been studied in the literature are
 called finite state machines. These mathematical models consist of states and
-and responses to actions from one state to another. In the context of the IPD, a
-finite state machine, is a mapping from states and opponent actions to states
-and an action. For further details on this type of strategy, the reader is
-referred to~\cite{Ashlock2006, Harper2017}.
+and responses to actions which indicate a next state given an action.  In the
+context of the IPD, a finite state machine, is a mapping from an arbitrary list
+of states and opponent actions (Cooperation or Defection) to states and an
```
Comment (Nikoleta-v3) [inline main.tex]: >>> Each player chooses simultaneously and independently between cooperation (C) or defection (D).

Use `C` or `D` or all low case to be consistent?

File: main.tex [hunk 6]
```diff
@@ -1541,7 +1541,7 @@ \subsubsection*{The effect of population size}
 \end{itemize}
 
 The results for \(x_{N/2}\) show similarities to the results for \(x_{N-1}\) and
-in particular TF1, TF2 and TF3 ranked one, three and eight. This is to be
+in particular TF1, TF2 and TF3 ranked one, third and eighth. This is to be
```
Comment (Nikoleta-v3) [inline main.tex]: first

**Comment by drvinceknight** [comment]
Thanks @Nikoleta-v3 and @marcharper, I've pushed minor tweaks.

> Let's make sure we have the appropriate Ashlock et al references since they go into much more detail.

@marcharper I've had a double check and I believe it's an appropriate Ashlock reference. It's pointing at "Changes in prisoners dilemma strategies over evolutionary time with different population sizes." 2006 which includes a good description of FSMs in the experiments section. Let me know if you had another one in mind.

**Comment by drvinceknight** [comment]
> Let me know if you had another one in mind.

(We can always list multiple ones hey? :+1:)

**Comment by marcharper** [comment]
That one is good. There's also "Why Some Representations Are More Cooperative Than Others For Prisoner's Dilemma" by W. Ashlock which discusses finite state machines, lookup tables, and neural networks, and a book by D. Ashlock that is much more expository and covers finite state representations: "Evolutionary Computation for Modeling and Optimization".

**Comment by drvinceknight** [comment]
Great call, I hadn't seen those and they look great, have added. I'll nudge @meatballs just to make sure we're all happy and aim to submit today.

**Comment by drvinceknight** [comment]
I've spoken to @meatballs and he's happy for us to proceed: I'm merging and resubmitting.

**Pull request merged**